src/nbl/asset/interchange/CSTLMeshFileLoader.cpp

```diff
@@ -24,6 +24,106 @@ constexpr auto COLOR_ATTRIBUTE = 1;
 constexpr auto UV_ATTRIBUTE = 2;
 constexpr auto NORMAL_ATTRIBUTE = 3;
 
+CSTLMeshFileLoader::CSTLMeshFileLoader(asset::IAssetManager* _m_assetMgr)
+	: IRenderpassIndependentPipelineLoader(_m_assetMgr), m_assetMgr(_m_assetMgr)
+{
+	initialize();
```

Comment:
stop calling `initialize` in the constructor

Reply:
corrected in following commit